README.md

```diff
@@ -1,113 +1,32 @@
-<p align="center">
-  <a href="https://www.gatsbyjs.com">
-    <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
-  </a>
-</p>
-<h1 align="center">
-  Starter for a Gatsby Plugin
-</h1>
+# gatsby-source-apostrophe
 
-A minimal boilerplate for the essential files Gatsby looks for in a plugin.
+Source plugin for accessing ApostropheCMS content APIs in Gatsby
 
-## 🚀 Quick start
+## Install
 
-To get started creating a new plugin, you can follow these steps:
+`npm install gatsby-source-apostrophe`
 
-1. Initialize a new plugin from the starter with `gatsby new`
```

Comment:
Should probably have a single note up front about where you learn to set up gatsby, although it shouldn't spend any more time on it than that.

Reply:
Since this is a Gatsby plugin, not an Apostrophe module, you'll already need a Gatsby site for this. I'll link to Gatsby's docs, but directing people to create a Gatsby app seems backwards.

I used existing source plugins as a model. https://www.gatsbyjs.com/plugins/gatsby-source-shopify/?=source-plugin

Reply:
I guess that's cool, then when we talk about this module and link to it we need to do a good job of providing context if the context ("why" and the broader "how" of apostrophe plus gatsby) isn't going to live in the source plugin docs.